ydb/core/health_check/health_check.cpp

```diff
@@ -3270,8 +3277,14 @@ class THealthCheckService : public TActorBootstrapped<THealthCheckService> {
         Become(&THealthCheckService::StateWork);
     }
 
+    void Handle(NConsole::TEvConsole::TEvConfigNotificationRequest::TPtr& ev) {
+        const auto& record = ev->Get()->Record;
+        HealthCheckConfig.CopyFrom(record.GetConfig().GetHealthCheckConfig());
```

Comment:
```suggestion
        if (record.GetConfig().HasHealthCheckConfig()) {
            HealthCheckConfig.CopyFrom(record.GetConfig().GetHealthCheckConfig());
        }
```

In the future, when (perhaps) there will be subscriptions to other kind of configurations, this piece will not need to be rewritten.